app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1196,7 +1196,11 @@ public function sendAction($objectId)
 
         $action   = $this->generateUrl('mautic_email_action', ['objectAction' => 'send', 'objectId' => $objectId]);
         $pending  = $model->getPendingLeads($entity, null, true);
-        $form     = $this->get('form.factory')->create(BatchSendType::class, [], ['action' => $action]);
+        $data     = [
+            'action' => $action,
+            'data'   => ['batchlimit' => $this->coreParametersHelper->get('mailer_memory_msg_limit')],
```

Comment:
I think that the changes in this controller should be reverted and instead change the hard-coded `100` to load the default value from the CoreParametersHelper in https://github.com/mautic/mautic/blob/4.x/app/bundles/EmailBundle/Form/Type/BatchSendType.php#L13

Reply:
@volha-pivavarchyk I agree with @escopecz

Reply:
done